src/contracts/pods/EigenPod.sol

```diff
@@ -793,6 +792,7 @@ contract EigenPod is IEigenPod, Initializable, ReentrancyGuardUpgradeable, Eigen
         return _validatorPubkeyHashToInfo[pubkeyHash].status;
     }
 
+    ///@notice Returns the validatorInfo for a given validatorPubkey
```

Comment:
view other functions for natspec standard

Reply:
to be fair our "natspec standard" is random af, some functions have descriptions for parameters, others don't, its a mess

Reply:
compromise to put the space inbetween / and @? it's making me cry.

Reply:
fixed lmao